src/Sylius/Bundle/CoreBundle/Fixture/CountryFixture.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\CoreBundle\Fixture;
+
+use Doctrine\Common\Persistence\ObjectManager;
+use Sylius\Bundle\FixturesBundle\Fixture\AbstractFixture;
+use Sylius\Component\Addressing\Model\CountryInterface;
+use Sylius\Component\Resource\Factory\FactoryInterface;
+use Symfony\Component\Config\Definition\Builder\ArrayNodeDefinition;
+use Symfony\Component\Intl\Intl;
+
+/**
+ * @author Kamil Kokot <kamil.kokot@lakion.com>
+ */
+final class CountryFixture extends AbstractFixture
+{
+    /**
+     * @var FactoryInterface
+     */
+    private $countryFactory;
+
+    /**
+     * @var ObjectManager
+     */
+    private $countryManager;
+
+    /**
+     * @param FactoryInterface $countryFactory
+     * @param ObjectManager $countryManager
+     */
+    public function __construct(
+        FactoryInterface $countryFactory,
+        ObjectManager $countryManager
+    ) {
+        $this->countryFactory = $countryFactory;
+        $this->countryManager = $countryManager;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function load(array $options)
+    {
+        $countriesCodes = array_keys(Intl::getRegionBundle()->getCountryNames());
+
+        foreach ($countriesCodes as $countryCode) {
+            /** @var CountryInterface $country */
+            $country = $this->countryFactory->createNew();
+            $country->setCode($countryCode);
+
+            $this->countryManager->persist($country);
+        }
+
+        $this->countryManager->flush();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getName()
+    {
+        return 'countries';
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configureOptionsNode(ArrayNodeDefinition $optionsNode)
+    {
+        // no options
```

Comment:
There's still an issue if some country codes would be used by Province / Zone fixtures while these countries wouldn't exist in the database.